.github/workflows/main.yml

```diff
@@ -321,6 +321,7 @@ jobs:
 
       - name: Zip log files on failure
         if: ${{ failure() }}
+        timeout-minutes: 1 # timeout after 1 minute
```

Comment:
Wasn't sure if 1 minute was a bit too short, but it looks like this normally completes within 10 seconds.

Reply:
yeah it's like now or never😂